f2/src/forms/WhatHappenedForm.js

```diff
@@ -28,45 +19,61 @@ export const WhatHappenedForm = (props) => {
 
   let formLabel = <Trans id="whatHappenedPage.summary" />
   let formHelpText = <Trans id="whatHappenedPage.hint" />
-  let nextButton = <Trans id="whatHappenedPage.nextPage" />
+  let nextPage = <Trans id="whatHappenedPage.nextPage" />
   let formHelpText2
 
   if (fyiForm) {
     formLabel = <Trans id="whatHappenedPage.fyi.summary" />
     formHelpText = <Trans id="whatHappenedPage.fyi.hint" />
     formHelpText2 = <Trans id="whatHappenedPage.fyi.hint2" />
-    nextButton = <Trans id="fyiForm.nextPage2" />
+    nextPage = <Trans id="fyiForm.nextPage2" />
   }
 
   return (
-    <Form
+    <Formik
       initialValues={whatHappened}
-      onSubmit={localOnSubmit}
-      render={({ handleSubmit }) => (
-        <Stack as="form" onSubmit={handleSubmit} shouldWrapChildren>
-          <FormLabel htmlFor="whatHappened">{formLabel}</FormLabel>
-          <FormHelperText htmlFor="whatHappened">
-            {formHelpText}
-            {formHelpText2 ? (
-              <p>
-                <br />
-                {formHelpText2}
-              </p>
-            ) : null}
-          </FormHelperText>
-          <Field
-            name="whatHappened"
-            component={TextArea}
-            h="300px"
-            marginTop="-1.5rem"
-          />
-          <NextAndCancelButtons
-            next={nextButton}
-            button={<Trans id="whatHappenedPage.nextButton" />}
-          />
-        </Stack>
+      onSubmit={(values) => {
+        props.onSubmit(values)
+      }}
+    >
+      {({ handleSubmit, handleChange, handleBlur }) => (
+        <Form onSubmit={handleSubmit}>
+          <Container>
+            <Row className="form-question">
+              <Row className="form-label">
+                {formLabel}
+              </Row>
+              <Row className="form-helper-text">
+                {formHelpText}
+                {formHelpText2 ? (
+                <p>
+                  <br />
```

Comment:
We could probably replace this with Jonh's <P> component from f2\src\components\formik\paragraph\index.js.

Reply:
This is from the old code (maybe yours:)) and it works well. May I ask the reason you want to replace it?

Reply:
No need to replace, just thought it would be a good time to use the <P> component with Emotion.styled. It's your call.